contracts/views/QiDaoView.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../actions/qidao/helpers/QiDaoHelper.sol";
+import "../interfaces/qidao/IStablecoin.sol";
+
+contract QiDaoView is QiDaoHelper {
+
+    function getVaultInfo(uint16 _vaultId, uint256 _userVaultId) public view returns (uint256 coll, uint256 debt) {
```

Comment:
Not just here but maybe in general don't use `_vaultId` as we are use to meaning a diff. thing in Maker. Instead something like qiVaultId, qiDaoVaultId, collVaultId, etc... something to indicate that it's a protocol wide id.